src/vstest.console/Internal/ConsoleLogger.cs

```diff
@@ -700,5 +706,13 @@ public static void RaiseTestRunWarning(string warningMessage)
 
             Output.Warning(AppendPrefix, warningMessage);
         }
+
+        /// <summary>
+        /// updates the test results
+        /// </summary>
+        public void UpdateTestsStatus()
```

Comment:
Can directly return the test status as a string. The field is not really used anywhere without first calling `UpdateTestsStatus()`.